src/Facebook/InstantArticles/Elements/Ad.php

```diff
@@ -80,6 +80,7 @@ public function disableDefaultForReuse()
      * Sets the height of your ad.
      *
      * @param int The height of your ad.
+     * @return $this
```

Comment:
Guess this is more <type> <what is inside> standard.
Can you add this, please?
`@return Ad the add self instance.

replicate this for all other with methods and getter ones.